components/eam/bld/namelist_files/namelist_defaults_eam.xml

```diff
@@ -11,7 +11,6 @@
 <dtime dyn="se"                    >1800</dtime>
 <dtime dyn="se"    hgrid="ne4np4"  >7200</dtime>
 <dtime dyn="se"    hgrid="ne11np4" >7200</dtime>
-<dtime dyn="se"    hgrid="ne16np4" >1800</dtime>
```

Comment:
Why did you delete this?

Reply:
Because the default is set to 1800 anyways. It actually seems like it should be set to something somewhat larger than 1800.

Reply:
It seems like it would be nice to have the default time step that automatically scales with the resolution... However, based on how the MMF responds to increasing the physics time step I think there's an underlying fundamental issue with this way of thinking. We have the tendency to let the max wind speed and CFL criteria to guide our time step choice, but it seems the time scale of signals on the smallest scales should really be the thing that guides this choice. This is obvious in the case of the MMF because the clouds are behaving the same at ne4 and ne30, so using the same time step ends up making sense. But I suppose we can extend this to the standard model too because the parameterized clouds and surface fluxes are still operating at the same time scales... So I guess my preference would be to have the ne30 time step be default for any coarser resolution, and let the CFL criteria dominate our time step choice for higher resolution. 

Anyway, that's a long-winded way to say that this change is fine with me  😄